workshops/1.md

```diff
@@ -57,64 +58,68 @@ We agreed on disabling GitHub issues for internal projects and moving all issue
 
 We'll include the project owner and project buddie to the README.
 
-We agreed that the README will be used to have a small description of what the project is about in non-techie language.
+We agree that the README will have a small description of what the project is about in non-technical language.
 
 ### GIFs
 
-Would be cool to have a better tool to make gifs or videos of our apps. We could provide a way to choose the type of device or simulator to use.
+Would be cool to have a better tool to make gifs or videos for our apps. We could provide a way to choose the type of device or simulator to use, amongst other things.
 
 ### Daily standups
 
-We agreed that daily standups suck, specially since they are not asynchronous.
+We agreed that daily standups suck, specially since they are not asynchronous. :+1:
 
 ### Workshop
 
-We said that the workshop was awesome, and we should try to have something similar once a month. Can't remember the date.
+We think that the workshop was awesome, and we should try to have something similar every on a smaller scale every so often.
 
 ### Articles
 
-We'll do articles, we talked about doing it in the simplest blogging platform we could find, the threshold to publish a new post should be super low, and basically help us to share fast and often.
+We want to write more articles. We talked about doing it in the simplest blogging platform we could find, so the threshold to publish a new post is super low, and basically help us to share fast and often.
 
-We'll make a new blog post about fun things that we learned, gotchas that we found while solving a problem, and posts introducing any open source component that we do.
+We'll make a new blog post about fun things that we learned, gotchas that we found while solving a problem, and posts introducing any open source component that we do. We'lll also write about some of the solutions that came out of this workshop, such as architectural guidelies for code, sample projects, etc.
 
 ### Gists
 
-We'll use gists more (?).
+We'll start use the gist feature from GitHUb more. It's a powerful tool to share snippets of code that delineate a problem, so we can quickly interate on it, without having to clone/build a whole project and get bogged down with details.
```

Comment:
No we didn't. Also copy/pasting a gist on a Playground is more efficient than passing a whole folder around.